sdk_contrib/async_processor/src/main/java/io/opentelemetry/sdk/contrib/trace/export/DisruptorEventQueue.java

```diff
@@ -138,19 +170,38 @@ void forceFlush() {
       if (isShutdown) {
         return;
       }
-      enqueue(null, EventType.ON_FORCE_FLUSH);
+      CountDownLatch flushLatch = new CountDownLatch(1);
+      enqueue(null, EventType.ON_FORCE_FLUSH, flushLatch);
+      try {
+        flushLatch.await();
+      } catch (InterruptedException e) {
+        // Preserve the interruption.
+        Thread.currentThread().interrupt();
+        logger.warning("Thread interrupted, shutdown may not finished.");
+      }
```

Comment:
Can shutdown be implemented as?
1. Set shutdown variable;
1. Flush and wait;
1. Call shutdown for the SpanProcessor next;

Reply:
Since the SpanProcessor is only visible in the `DisruptorEventHandler` class, we need to:
1. Set shutdown variable
2. Enqueue a Shutdown event
3. Wait for Shutdown event to be processed